kncaputo/underrated
# Feature/sort and filter

## What's this PR do?
- Add search bar for users to search movies by title
- Add filtering options to find movies by genre or rating
- Add trailer to banner that plays on main page load
- Reformat budget/revenue styling to show '-' instead of '$0'
- Add a clear button to clear filters 

## Where should the reviewer start?
- `Search.js`, `Banner.js`, and `App.js`

## How should this be manually tested?
- Open app and ensure trailer plays on page load
- Search for a movie by title 
- Search for a movie by genre or rating using the dropdowns
- Clear the filters by pressing the clear button
- View several movies to ensure that '-' shows up

## Any background context you want to provide?
- Additional styling for the banner needs to be added (i.e. movie info)

## Screenshots/GIFs
- <img width="1329" alt="Screen Shot 2021-01-03 at 9 19 10 AM" src="https://user-images.githubusercontent.com/58453566/103483492-d648c280-4da4-11eb-92c6-61a3dad37229.png">
- <img width="1323" alt="Screen Shot 2021-01-03 at 9 19 41 AM" src="https://user-images.githubusercontent.com/58453566/103483496-dba60d00-4da4-11eb-8c39-55bdac3c2806.png">
